desktop-src/direct3dhlsl/dx-graphics-hlsl-to-sample.md

```diff
@@ -157,6 +157,8 @@ VS_OUTPUT In;
 
 Texture sampling uses the texel position to look up a texel value. An offset can be applied to the position before lookup. The sampler state contains the sampling and filtering options. This method can be invoked within a pixel shader, but it is not supported in a vertex shader or a geometry shader.
 
+Use an offset only at an integer miplevel; otherwise, you may get results that do not translate well to hardware.
+
```

Comment:
as we discussed, I think this should be under Applying texture coordinate offsets